tests/_internal/test_resolve_target.py

```diff
@@ -119,3 +121,43 @@ def test_returns_contents_of_all_htm_files_in_given_directories(self):
             result = _get_all_htm_files_contents([pathlib.Path("/test-1"), pathlib.Path("/test-2")])
 
         self.assertEqual(result, ["foo", "bar"])
+
+
+class TestReadHtmFilesContents(TestCase):
+    def test_handles_unreadable_htm_file(self):
+        with Patcher() as patcher:
+            patcher.fs.create_file("mbed.htm", contents="foo")
+
+            result = _read_htm_file_contents([pathlib.Path("mbed.htm"), pathlib.Path("error.htm")])
+
+        self.assertEqual(result, ["foo"])
+
+
+class TestIsHtmFile(TestCase):
+    def test_lower_case_htm(self):
+        with Patcher() as patcher:
+            patcher.fs.create_file("mbed.htm", contents="foo")
```

Comment:
Correct me if I'm wrong, but since `_is_htm_file` operates on `pathlib.Path` only, I believe it's not necessary to patch the file system.

Reply:
I cannot correct you, these kind of evolved over time. Fixed now.